src/style.css

```
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 900;
+}
+
+.bigForm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.searchLabel {
+  position: relative;
+}
+
+.serachInput {
+  font-family: Avenir-Book, sans-serif;
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.serachInput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.serachInput:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.serachInput::placeholder {
+  color: #3d4e61;
+}
+
+.searchIcon {
+  box-sizing: border-box;
+  position: absolute;
+  left: 26px;
+  top: 0;
+}
+
+.big {
+  height: 70px;
+  font-size: 16px;
+  line-height: 21.86px;
```

don't copy past all styles from Figma.
There is no necessity to add hundredths in the values
```suggestion
  line-height: 22px;
```